vrouter.go

```diff
@@ -397,6 +480,78 @@ func (self *Vrouter) UpdateLocalEndpoint(endpoint *OfnetEndpoint, epInfo Endpoin
 	return nil
 }
 
+// AddHostPort sets up host access.
+func (self *Vrouter) AddHostPort(hp HostPortInfo) error {
+	if hp.Kind == "NAT" && self.hostNATInfo.PortNo != 0 {
+		log.Errorf("Host NAT port exists: %+v", self.hostNATInfo)
+		return fmt.Errorf("Host NAT port exists")
+	}
+
+	ipDa, daMask, err := ParseIPAddrMaskString(hp.IpAddr)
+	if err != nil {
+		log.Errorf("Bad host route - %v", err)
+		return err
+	}
+
+	netMask := pflag.ParseIPv4Mask(daMask.String())
+	maskedIP := ipDa.Mask(netMask)
+
+	// Save the info
+	self.hostNATInfo = hp
```

Comment:
As of now, no. But, it could be leveraged in the future to add a different kind of host port.